draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3348,21 +3350,20 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
            
         </list></t>
 
-        <section anchor="trigger-scope-capability-object" title="CI/T Trigger Scope Capability Object Serialization">
-          <t>
-             The following shows an example of a JSON serialized CI/T Trigger Scope Capability object serialization for
-             a dCDN that supports the preposition and invalidation of content, using "urls" and "ccids" 
-             Generic Spec types, with "time-policy" but only for the "preposition" action. Note that in this example, purge 
-             is not supported neigher any operation on metadata.
-          </t>
+        	<section anchor="trigger-scope-capability-object" title="CI/T Trigger Scope Capability Object Serialization">
+          	<t>
+             	The following shows an example of a JSON serialized CI/T Trigger Scope Capability object serialization for
+		a dCDN that supports the preposition and invalidation of content, using "urls" and "ccids" 
+             	Generic Spec types, with "time-policy" but only for the "preposition" action. Note that in this example, purge 
+             	is not supported neigher any operation on metadata.
```

Comment:
Not sure how either of us missed "neigher" in the original document.
Maybe:  "Note that in this example, the purge action is not supported, and no actions involving metadata are supported either."

Reply:
Well, apparently neigher is a legitimate word, one who neighs :)
Used your new wording